modules/apps/jaggery/emm/src/emm/units/user-listing/public/js/user-listing.js

```diff
@@ -139,4 +138,70 @@ $("a.remove-user-link").click(function () {
     $("a#remove-user-cancel-link").click(function () {
         hidePopup();
     });
+}
+
+/**
+ * Following on click function would execute
+ * when a user type on the search field on User Listing page in
+ * WSO2 MDM Console then click on the search button.
+ */
+
+$("#search-btn").click(function () {
+    var searchQuery = $("#search-by-username").val();
+    loadUsers(searchQuery);
```

Comment:
Recommended fixes can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/166